config/jobs-async-upload/jobs-async-upload-s3-to-oci-template.yaml

```diff
@@ -110,3 +113,7 @@ objects:
                   value: "${MODEL_SYNC_MODEL_VERSION_ID}"
                 - name: MODEL_SYNC_MODEL_ARTIFACT_ID
                   value: "${MODEL_SYNC_MODEL_ARTIFACT_ID}"
+                - name: MODEL_SYNC_SOURCE_S3_CREDENTIALS_PATH
+                  value: "/opt/creds/source"
```

Comment:
since this is a volume mount, defines the volume mount coordinate per requirement